ibis/backends/duckdb/tests/test_geospatial.py

```diff
@@ -195,3 +195,24 @@ def test_geospatial_buffer(zones, zones_gdf):
     gp_buffer = zones_gdf.geometry.buffer(100.0, resolution=8)
 
     gtm.assert_geoseries_equal(buffer.to_pandas(), gp_buffer, check_crs=False)
+
+
+def test_geospatial_convert(zones, zones_gdf):
+    # the zones data is in CRS: EPSG:2263
+    # let's transform to EPSG:4326 (latitude-longitude projection)
+
+    zones_geom_ll = zones.geom.convert("EPSG:2263", "EPSG:4326")
+
+    zones_gdf_ll = zones_gdf.geometry.to_crs(crs=4326)
+    # breakpoint()
+
+    # this doesn't work for all the points
+    # gtm.assert_geoseries_equal(zones_geom_ll.to_pandas(), zones_gdf_ll,  check_less_precise=True, check_crs=False)
+
+    # this will pass but it's very big tolerance
+    assert all(
```

Comment:
I don't like that to get the test to pass the tolerance even after using [`set_precision`](https://shapely.readthedocs.io/en/stable/reference/shapely.set_precision.html) is quite high (1e-2), The test is also a bit slow. 
I tried using the lines dataset, and I can get it to pass with `1e-3` but this si still not ideal. 
Alternatively we can do just a snapshot test. 

Any thoughts? 
cc: @gforsyth and @cpcloud

Reply:
Can we test against a hand-rolled table that we put in `duckdb.sql`, and also open an issue on `duckdb_spatial` about this?